src/tasks/AnomalyHunter.py

```
@@ -0,0 +1,344 @@
+        claimed_reward = False
+
+        def action(count):
+            nonlocal pending_reward_ready, claimed_reward
+            reward_found = pending_reward_ready
+            pending_reward_ready = False
+            if not reward_found:
+                reward_found = self.walk_to_bosstreasure()
+
+            claimed_reward = bool(reward_found)
+            if reward_found:
+                self.log_info("发现宝箱，正在领取交互中-测试中跳过点击")
```

Ruff (0.15.21)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)